src/Taboo/TurnManager.py

```diff
@@ -18,8 +22,10 @@
 )
 
 class TurnState(Enum):
+    GAME_START_WAIT = 0
```

Comment:
Are GAME_START_WAIT and GAME_OVER really TurnStates?
I can understand the purpose.

Turn when instantiated is in KICKOFF_WAIT, then it goes to RUNNING and then OVER. Isn't this enough.

Reply:
These are perhaps turn manager states. I am happy to rename the Enum to TurnMgrState if you think that's appropriate.

The distinction between KICKOFF_WAIT and GAME_START_WAIT to make updateState() more robust. We should only send the KICKOFF_WAIT message after all players are in READY state/startNewTurn() has been called the first time.

Your thoughts?

Reply:
Yeah, these can be seen as TurnMgr states. 

For a particular Turn instance, if you want to "archive" it, something can be added there.